### ISSUE TITLE
Timeout error right after execution

### ISSUE THREAD

**Issue by rene78** [drolbr/Overpass-API#718]
A timeout error is returned after execution of this query:

`//overpass-api.de/api/interpreter?data=[adiff:"2024-03-15T07:14:05.686Z"][bbox:47.64116208454375,9.45399284362793,47.6942810889081,9.496135711669922][out:xml];nw;out geom meta;`

The XML with the following error message is returned:

`"runtime error: Query timed out in "query" at line 1 after 593784 seconds."`

The error is returned immediately, not after "593784 seconds".

If a smaller bounding box is used it works again. Do I need increase the [maxsize](https://dev.overpass-api.de/overpass-doc/en/preface/commons.html#Timeout%20and%20Maxsize:~:text=code%20429.-,Timeout%20and%20Maxsize,-Independent%20of%20the) parameter to avoid the issue?

The problem appeared while using [OSM Latest Changes](https://rene78.github.io/latest-changes/#14/47.6677/9.4751) with a relatively small bounding box.

Below is the whole XML that is beeing returned from overpass.
```
<?xml version="1.0" encoding="UTF-8"?>
<osm version="0.6" generator="Overpass API 0.7.62 7c32a7d5">
<note>The data included in this document is from www.openstreetmap.org. The data is made available under ODbL.</note>
<meta osm_base="2024-03-22T07:14:44Z"/>

  <bounds minlat="47.6411621" minlon="9.4539928" maxlat="47.6942811" maxlon="9.4961357"/>

<remark> runtime error: Query timed out in "query" at line 1 after 593784 seconds. </remark>

</osm>
```

**Comment by mmd-osm** [comment]
> Do I need increase the maxsize  parameter to avoid the issue?

To me this looks more like a regression in version 0.7.62, which has been released only a couple of days ago.

**Comment by drolbr** [comment]
Thank you for the bug report.

The bug is fixed in 084b4234b4ceb54bc346f10354a12a1365d8f7c2 and thus in release 0.7.62.1.
